tests/configs-test/usersecrets_slightfile.toml

```diff
@@ -0,0 +1,20 @@
+specversion = "0.1"
+
+[[secret_settings]]
+name = "THIS_IS_ANOTHER_TEST_CONFIG_1061408"
+value = "m7N0Muoz0ICs7Owemrg"
+
+[[secret_settings]]
+name = "THIS_IS_ANOTHER_TEST_CONFIG_1061296"
+value = "g88GVRuP1o2aqE6L1Mg"
+
+[[secret_settings]]
+name = "THIS_IS_ANOTHER_TEST_CONFIG_1061248"
```

Comment:
I've noticed that each run of config adds a new secret_settings to the slightfile. How can we avoid this?